.clang-tidy

```diff
@@ -5,7 +5,7 @@ concurrency-*,-modernize-use-trailing-return-type, -modernize-use-nodiscard,
 -readability-redundant-access-specifiers,-readability-qualified-auto,
 -cppcoreguidelines-avoid-non-const-global-variables,-cppcoreguidelines-owning-memory,
 -readability-convert-member-functions-to-static,-bugprone-easily-swappable-parameters,
--cppcoreguidelines-pro-type-static-cast-downcast'
+-cppcoreguidelines-pro-type-static-cast-downcast, -readability-uppercase-literal-suffix'
```

Comment:
Why is `readability-uppercase-literal-suffix` disabled now? In the recent months I have already moved many f's to F's for floating point numbers.

For other types like long or unsigned long the suffixes L or UL are more common than l or ul and IMO the uppercase forms should be enforced.

Reply:
It seemed to me to be just noise without any real benefit, but I don't feel strongly about it -- I certainly agree that for L and UL uppercase is far more readable. For f vs F I really don't care, so I have not been asking developers to change it. I can start doing so if you'd like, and leave this linter warning in there.